### PR TITLE
add content_type in <.publish.json>

### PR DESCRIPTION
**Pull request by YunshuMeg**
[AB#700644](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/700644)

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/docfx?pullRequest=8163) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docfx?pullRequest=8163) before we can accept your contribution.<br/><br/>:x: YunshuMeg [sign now](https://cla.dotnetfoundation.org/dotnet/docfx?pullRequest=8163)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docfx?pullRequest=8163) it.</sub>

**Review by 928PJY — APPROVED** [review]

**Review by freewheel70 — APPROVED** [review]
LGTM

File: src/docfx/publish/PublishItem.cs [hunk 1]
```diff
@@ -30,6 +30,8 @@ internal class PublishItem
     /// </summary>
     public string? SourcePath { get; init; }
 
+    public string? ContentType { get; init; }
```
Comment (AngryBerryMS) [inline src/docfx/publish/PublishItem.cs:33]: Should it nullable?

Comment (928PJY) [reply src/docfx/publish/PublishItem.cs:33]: I notice all the property under this class are nullable, @AngryBerryMS do you know the background?

**Pull request merged**